### PR TITLE
feat(statics): add ERC20 tokens

### PR DESCRIPTION
**Pull request by Joelsx**
add ERC20 tokens

BREAKING CHANGE: modfiy symbol 'seth-h' to 'Staked ETH Harbour' and ofcholo to ocfhot

BG-53319

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -739,7 +746,7 @@ export const coins = CoinMap.fromCoins([
   erc20('tlab', 'TLAB', 18, '0x36222730e78a8656230c54f9e402a062d168d2d9', UnderlyingAsset.TLAB),
   erc20('tnt', 'Tierion', 8, '0x08f5a9235b08173b7569f83645d2c7fb55e8ccd8', UnderlyingAsset.TNT),
   erc20('tok', 'Tokenplace', 8, '0x4fb721ef3bf99e0f2c193847afa296b9257d3c30', UnderlyingAsset.TOK),
-  erc20('trac', 'Trace', 18, '0xaa7a9ca87d3694b5755f213b5d04094b8d0f0a6f', UnderlyingAsset.TRAC),
+  erc20('trac', 'OriginTrails', 18, '0xaa7a9ca87d3694b5755f213b5d04094b8d0f0a6f', UnderlyingAsset.TRAC),
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:749]: Breaking change needs to be documented in the commit footer.

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -712,7 +719,7 @@ export const coins = CoinMap.fromCoins([
   erc20('stbu', 'Stobox Token', 18, '0x212dd60d4bf0da8372fe8116474602d429e5735f', UnderlyingAsset.STBU),
   erc20('stc', 'Student Coin', 2, '0xb8b7791b1a445fb1e202683a0a329504772e0e52', UnderlyingAsset.STC),
   erc20('stcv2', 'Student Coin V2', 18, '0x15b543e986b8c34074dfc9901136d9355a537e7e', UnderlyingAsset.STCV2),
-  erc20('seth-h', 'Staked Aave', 18, '0x65077fa7df8e38e135bd4052ac243f603729892d', UnderlyingAsset['SETH-H']),
+  erc20('seth-h', 'Staked ETH Harbour', 18, '0x65077fa7df8e38e135bd4052ac243f603729892d', UnderlyingAsset['SETH-H']),
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:722]: Breaking change needs to be documented in the commit footer.

File: modules/statics/src/coins.ts [hunk 3]
```diff
@@ -889,12 +898,17 @@ export const coins = CoinMap.fromCoins([
   ofcerc20('ofcftm', 'Fantom Token', 18, UnderlyingAsset.FTM),
   ofcerc20('ofcgala', 'Gala', 8, UnderlyingAsset.GALA),
   ofcerc20('ofcgno', 'Gnosis', 18, UnderlyingAsset.GNO),
-  ofcerc20('ofcholo', 'Holo Token', 18, UnderlyingAsset.HOT),
+  ofcerc20('ofchot', 'Holo Token', 18, UnderlyingAsset.HOT),
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:901]: Breaking change needs to be documented in the commit footer.

**Review by mmcshinsky-bitgo — CHANGES_REQUESTED** [review]

File: modules/statics/src/coins.ts [hunk 4]
```diff
@@ -329,6 +331,7 @@ export const coins = CoinMap.fromCoins([
   erc20('cel', 'Celsius', 4, '0xaaaebe6fe48e54f431b0c390cfaf0b017d09d42d', UnderlyingAsset.CEL),
   erc20('celr', 'Celer Network', 18, '0x4f9254c83eb525f9fcf346490bbb3ed28a81c667', UnderlyingAsset.CELR),
   erc20('ceth', 'Compound Ether', 8, '0x4ddc2d193948926d02f9b1fe9e1daa0718270ed5', UnderlyingAsset.CETH),
+  erc20('cho', 'Choise.com', 8, '0xbba39fd2935d5769116ce38d46a71bde9cf03099', UnderlyingAsset.CHO),
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:334]: Same with this one. It's listed as lowercase `choise.com Token`.

File: modules/statics/src/coins.ts [hunk 5]
```diff
@@ -388,6 +391,7 @@ export const coins = CoinMap.fromCoins([
   erc20('dgd', 'Digix DAO', 9, '0xe0b7927c4af23765cb51314a0e0521a9645f0e2a', UnderlyingAsset.DGD),
   erc20('dgx', 'Digix', 9, '0x4f3afec4e5a3f2a6a1a411def7d7dfe50ee057bf', UnderlyingAsset.DGX),
   erc20('digg', 'Digg', 9, '0x798d1be841a82a273720ce31c822c61a67a601c3', UnderlyingAsset.DIGG),
+  erc20('dia', 'DIA Token', 9, '0x84ca8bc7997272c7cfb4d0cd3d55cd942b3c9419', UnderlyingAsset.DIA),
```
Comment (mmcshinsky-bitgo) [inline modules/statics/src/coins.ts:394]: This one is listed as a single word `DIAToken`.

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (mmcshinsky-bitgo) [reply modules/statics/src/coins.ts:749]: Does this one need to change? The Token Tracker has it listed as Trace.

File: modules/statics/src/coins.ts [hunk 6]
```diff
@@ -312,6 +313,7 @@ export const coins = CoinMap.fromCoins([
   erc20('burp', 'Big Town Chef', 18, '0x33f391f4c4fe802b70b77ae37670037a92114a7c', UnderlyingAsset.BURP),
   erc20('busd', 'Binance USD', 18, '0x4fabb145d64652a948d72533023f6e7a623c7c53', UnderlyingAsset.BUSD),
   erc20('buy', 'buying.com', 18, '0x0d7f0fa3a79bfedbab291da357958596c74e27d7', UnderlyingAsset.BUY),
+  erc20('bpt', 'BlackPool', 18, '0x0ec9f76202a7061eb9b3a7d6b59d36215a7e37da', UnderlyingAsset.BPT),
```
[the remark this answers is not in the document]
Comment (Joelsx) [reply modules/statics/src/coins.ts:316]: Probably, I will since thats what etherscan said.

File: modules/statics/src/coins.ts (hunk 4 above)
Comment (Joelsx) [reply modules/statics/src/coins.ts:334]: ok

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (Joelsx) [reply modules/statics/src/coins.ts:749]: was request in this ticket https://bitgoinc.atlassian.net/browse/BG-54049

**Comment by mmcshinsky-bitgo** [comment]
Thanks! LGTM.

**Comment by mmcshinsky-bitgo** [comment]
Thanks for the updates! Looks like we need the breaking changes documented in the commit footer and then we are good to go: https://www.conventionalcommits.org/en/v1.0.0/

**Comment by mmcshinsky-bitgo** [comment]
Remove the line between BREAKING CHANGE and BG-<> and you are good to go: https://www.conventionalcommits.org/en/v1.0.0/

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Review by mmcshinsky-bitgo — DISMISSED** [review]

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**